### DISCUSSION TITLE
Allow Changing the Title of Currently Open Windows Using Window::current()

### DISCUSSION THREAD

**Discussion by molbal** [NativePHP/laravel#329] [Ideas]
I am writing to request an enhancement to the Window::current() function. Currently, Window::current() provides access to the window that is presently open. However, there is no straightforward way to change the title of this window once it has been created.

**Proposed Enhancement:**
Add a method to `Window::current()` that allows changing the title of the currently open window. 

Example: 
```php
$window = Window::current()->title('New title');
```

Cheers

**Comment by simonhamp** [comment]
You can [now](https://github.com/NativePHP/laravel/pull/391) use the `Window::title()` method to update a window's URL.
